### PR TITLE
update to 3.34 version of the GNOME platform

### LINKED ISSUE #3: Update runtime to 3.34

### PR DESCRIPTION
**Pull request by gdesmott**
Fix #3

### CODE REVIEW

**Comment by gdesmott** [comment]
bot, build

File: org.karapulse.Karapulse.json [hunk 1]
```diff
@@ -75,6 +75,31 @@
                 "./install.sh --prefix=/app/sdk/rust --disable-ldconfig --verbose"
             ]
         },
+        {
+            "name": "python-2.7",
```
Comment (LeandroStanger) [inline org.karapulse.Karapulse.json]: Add Shared Modules
https://github.com/flathub/flathub/wiki/App-Requirements#shared-modules
https://github.com/flathub/shared-modules/tree/master/python2.7

Comment (gdesmott) [reply org.karapulse.Karapulse.json]: @bochecha suggested me to copy it so I can use ` "cleanup": ["*"] ` as I just need python for the build and not in the package.

**Comment by gdesmott** [comment]
The package build fine now.

But as `html5 codecs` as been removed in the newer runtime I now have to use `openh264` for video decoding which isn't working: https://gitlab.gnome.org/GNOME/gnome-build-meta/issues/204

**Comment by TingPing** [comment]
You can remove all DConf permissions also (did you ever even use DConf?).

**Comment by gdesmott** [comment]
Indeed I'm not using it yet. Removing.

**Pull request merged**